samples/property/provider/src/main.rs

```diff
@@ -2,28 +2,24 @@
 // Licensed under the MIT license.
 // SPDX-License-Identifier: MIT
 
-mod provider_impl;
-
 use digital_twin_model::{sdv_v1 as sdv, Metadata};
 use env_logger::{Builder, Target};
 use log::{debug, info, warn, LevelFilter};
-use parking_lot::{Mutex, MutexGuard};
+use paho_mqtt as mqtt;
 use samples_common::constants::{digital_twin_operation, digital_twin_protocol};
-use samples_common::utils::{retrieve_invehicle_digital_twin_url, retry_async_based_on_status};
 use samples_common::provider_config;
+use samples_common::utils::{retrieve_invehicle_digital_twin_url, retry_async_based_on_status};
 use samples_protobuf_data_access::digital_twin::v1::digital_twin_client::DigitalTwinClient;
-use samples_protobuf_data_access::digital_twin::v1::{EndpointInfo, EntityAccessInfo, RegisterRequest};
-use samples_protobuf_data_access::sample_grpc::v1::digital_twin_consumer::digital_twin_consumer_client::DigitalTwinConsumerClient;
-use samples_protobuf_data_access::sample_grpc::v1::digital_twin_consumer::PublishRequest;
-use samples_protobuf_data_access::sample_grpc::v1::digital_twin_provider::digital_twin_provider_server::DigitalTwinProviderServer;
+use samples_protobuf_data_access::digital_twin::v1::{
+    EndpointInfo, EntityAccessInfo, RegisterRequest,
+};
 use serde_derive::{Deserialize, Serialize};
-use std::collections::HashSet;
-use std::net::SocketAddr;
-use std::sync::Arc;
+use tokio::signal;
 use tokio::time::{sleep, Duration};
-use tonic::{Status, transport::Server};
+use tonic::Status;
 
-use crate::provider_impl::{ProviderImpl, SubscriptionMap};
+const MQTT_CLIENT_ID: &str = "property-subscriber";
+const MQTT_QOS: i32 = 1;
```

Comment:
Instead of defining your own QOS constant, you can use paho_mqtt::types::QOS_1 (https://docs.rs/paho-mqtt/latest/paho_mqtt/types/constant.QOS_1.html).